repos/fdbt-netex-output/src/data/auroradb.ts

```diff
@@ -77,7 +77,39 @@ export const getOperatorDataByNocCode = async (nocCodes: string[]): Promise<Oper
             throw new Error(`No operator data found for nocCodes: ${cleansedNocs.join(',')}`);
         }
 
-        return operatorData;
+        const allNocCodes: string[] = [];
+        let nocCount: { [key: string]: number } = {};
+
+        for (let i = 0; i < operatorData.length; i++) {
```

Comment:
Usually, we should be using a .forEach or `for (const operator of operatorData)` instead of a for loop like this.